vspace/.gitignore

```diff
@@ -1,4 +1,5 @@
 /target/
 /Users/
 /.settings/
-.DS_Store
+/.DS_Store
```

Comment:
I believe this will not ignore all .DS_Store files that are not in the root folder.